acasclient/acasclient.py

```diff
@@ -440,6 +440,25 @@ def add_file_to_lot(self, lot_corp_name, file, file_type, writeup=None):
         """
         return self.add_files_to_lot(lot_corp_name, [{"file": file, "file_type": file_type, "writeup": writeup}])
 
+    def _get_cmpdreg_files_content(self, files):
+        """Get the content of cmpdreg files
+
+        Pass an array of cmpdreg_files to download and fill their content attribute
+
+        Args:
+            files: An array of cmpdreg_files to download in the format
```

Comment:
I am not sure what a good name would be, but `files` doesn't seem to totally indicate what kind of  a data structure it is. I would think this would just a take list of file paths, but it's more than that - it's a list of dictionaries which contain the "url" key(maybe use type annotation to force users to read the documentation?).

Alternatively we could just keep this method to take an argument called: `urls` and return a list of dictionaries with `url` and `content` keys.

Reply:
Thanks, point taken, it was confusing. Considering this is supposed to be the lot.fileList attribute, I just described it as that and anyone using it can see the structure that is supposed to be passed in now.